doc/man3/EVP_PKEY_set1_RSA.pod

```diff
@@ -15,6 +15,16 @@ EVP_PKEY_set1_engine, EVP_PKEY_get0_engine - EVP_PKEY assignment functions
 
  #include <openssl/evp.h>
 
+ int EVP_PKEY_id(const EVP_PKEY *pkey);
+ int EVP_PKEY_base_id(const EVP_PKEY *pkey);
+ int EVP_PKEY_type(int type);
+
+Deprecated since OpenSSL 3.0, can be hidden entirely by defining
```

Comment:
Does this refer to the function above or below the statement.. I know but maybe make it clear.

Reply:
This is boilerplate wording with 71 other matches in current master according to my grep. If we want to change it I suggest doing it in a separate PR rather than this one place having different wording.